.github/workflows/wheels.yml

```diff
@@ -1,6 +1,14 @@
 name: Wheels
 
 on:
+  schedule:
+  #        ┌───────────── minute (0 - 59)
+  #        │  ┌───────────── hour (0 - 23)
+  #        │  │ ┌───────────── day of the month (1 - 31)
+  #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+  #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+  #        │  │ │ │ │
+  - cron: "42 1 * * 0,3"
```

Comment:
The wheels build takes up a fair amount of CI time/runners.

@radarhere Is 01:42 UTC going to interfere with when you're active? We can move it to your night/inactive time if you like.

Reply:
Thanks for the thought, but that's not a problem for me personally - especially since https://github.com/python-pillow/Pillow/pull/8236/commits/d83c7b38c42952dbd3fcab1807732363d6e64607 skips the aarch64 jobs.